// imports 

const main = async () => {

}

main().then(() => process.exit(0))
.catch((error) => {
    console.log(error);
})